docs/conf.py

```diff
@@ -115,7 +115,6 @@
 nbsphinx_timeout = 360
 nbsphinx_execute = os.getenv("QISKIT_DOCS_BUILD_TUTORIALS", "never")
 nbsphinx_widgets_path = ""
-exclude_patterns = ["_build", "**.ipynb_checkpoints"]
```

Comment:
Why are we removing this? I don't object, I would just like to understand :smile:

Reply:
This is a duplicate entry :-) There is already the same statement below.